tests/functional/ovncontroller_controller_test.go

```diff
@@ -335,6 +352,137 @@ var _ = Describe("OVNController controller", func() {
 			Expect(th.GetConfigMap(scriptsCM).Data["net_setup.sh"]).Should(
 				ContainSubstring("addr show dev %s", ovncontroller.Spec.NetworkAttachment))
 		})
+		It("should create an external ConfigMap with expected key-value pairs", func() {
+			internalAPINADName := types.NamespacedName{Namespace: namespace, Name: "internalapi"}
+			nad := th.CreateNetworkAttachmentDefinition(internalAPINADName)
+			DeferCleanup(th.DeleteInstance, nad)
+			externalCM := types.NamespacedName{
+				Namespace: OVNControllerName.Namespace,
+				Name:      fmt.Sprintf("%s-%s", OVNControllerName.Name, "config"),
+			}
+
+			daemonSetName := types.NamespacedName{
+				Namespace: namespace,
+				Name:      "ovn-controller",
+			}
+			SimulateDaemonsetNumberReadyWithPods(
+				daemonSetName,
+				map[string][]string{namespace + "/internalapi": {"10.0.0.1"}},
+			)
+			externalSBEndpoint := "10.0.0.254"
+			SetExternalEndpoint(dbs[1], externalSBEndpoint)
+
+			Eventually(func() corev1.ConfigMap {
+				return *th.GetConfigMap(externalCM)
+			}, timeout, interval).ShouldNot(BeNil())
+
+			Expect(th.GetConfigMap(externalCM).Data["ovsdb-config"]).Should(
+				ContainSubstring("ovn-remote=%s", externalSBEndpoint))
+			Expect(th.GetConfigMap(externalCM).Data["ovsdb-config"]).Should(
+				ContainSubstring("ovn-encap-type=%s", "geneve"))
+
+			// TODO(ihar) shouldn't these somehow get "default" values from kubebuilder?
```

Comment:
we need to fix CreateOVNDBClusters to use CreateUnstructured to get it work, /me will send the patch

Reply:
I assume this is https://github.com/openstack-k8s-operators/ovn-operator/pull/112

If it lands before this PR, I will rebase.